SEED-platform/seed
# 1867 bldg parsing

#### Any background context you want to provide?
Bldg and Building were not accurately handled in the address names. 

#### What's this PR do?
* Handle subaddress types in the address normalization
* Reruns the normalize_address on all properties and tax lots. The migration may take a while based on the size of your database.
* upgrade streetaddress dependency.

#### How should this be manually tested?
* import these data ([test_data.xlsx](https://github.com/SEED-platform/seed/files/3143565/test_data.xlsx)) 
* Map concatenated field to address_line_1
* Ensure that the SEELY addresses are individual records

#### What are the relevant tickets?
#1867 

**NOTE: Merge #1866 first**

#### Screenshots (if appropriate)
